corehq/messaging/smsbackends/twilio/models.py

```diff
@@ -45,11 +51,24 @@ def get_pass_through_opt_in_keywords(cls):
     def get_opt_out_keywords(cls):
         return ['STOP', 'STOPALL', 'UNSUBSCRIBE', 'CANCEL', 'END', 'QUIT']
 
+    def _convert_to_whatsapp(self, number):
+        if WHATSAPP_PREFIX not in number:
```

Comment:
i am not clear why this would already have the prefix in the number?
We are using this internally, are we saying that users can also type their number with whatsapp prefix?

Reply:
I don't expect this block to execute, this is just to make the code more error-tolerant.

Reply:
the error would be that a number is passed with the prefix already present, so that would be an error and we should raise an error instead of silently play along unless we want to support that as a feature?

Reply:
Switched to asserts in ef05eae0dd2